.github/README.md

```diff
@@ -14,7 +14,7 @@ There are two types of content, each of which has its own challenges for transla
   _e.g. "Please enter a valid email address."_
 
   Django already provides a
-  [solution](https://docs.djangoproject.com/en/2.1/topics/i18n/translation/)
+  [solution](https://docs.djangoproject.com/en/3.1/topics/i18n/translation/)
```

Comment:
I this this should remain the lowest possible version to ensure backwards-compatibility.
So `2.2` for now.